### PR TITLE
Add testing for all util and model

### PR DESCRIPTION
**Pull request by heug**
Also add unit tests for configComments

### CODE REVIEW

**Comment by makotom** [comment]
The error in CI is due to cURL version? Anyway adding `--http1.1` for the curl command should "solve" the issue. 😢

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
 
 jobs:
-  build:
+  build-and-unit-test:
+    docker:
+      - image: circleci/node:12.8.0
+    working_directory: ~/project
+    steps:
+      - checkout
+      - run:
```
Comment (makotom) [inline .circleci/config.yml]: [It's optional] No dependency cache? 🤣

File: .circleci/config.yml [hunk 2]
```diff
@@ -20,15 +33,15 @@ jobs:
...
+  
+  output-smoke-test:
     docker:
       - image: circleci/node:12.8.0
     working_directory: ~/project
     steps:
       - run:
           name: install CircleCI CLI
-          command: curl -fLSs https://circle.ci/cli | sudo bash
+          command: sudo curl -fLSs https://circle.ci/cli | sudo bash
```
[the remark this answers is not in the document]
Comment (heug) [reply .circleci/config.yml]: fixed! works like a charm.

[the remark this answers is not in the document]
Comment (heug) [reply .circleci/config.yml]: removed `sudo`

File: .circleci/config.yml (hunk 1 above)
Comment (heug) [reply .circleci/config.yml]: added :P

**Pull request merged**